### ISSUE TITLE
How to define training set and test set? Is it randomly generated? 

### ISSUE THREAD

**Issue by 19991105** [dunbar12138/DSNeRF#108]
Hi, Your work is so great! But I have a questionn for you. You took all datasets as colmap_llff datasets rather than llff datasets for training. Is that so? So how did you define which images were training set and which were test set?
